------------
05-Sep-2024 12:46:06 
 /Users/eaguerov/Documents/Brown/impacts/km-dropplet-solidsubstrate-v3/matlab/1_code/sweeper_experiments
if system_dependent('IsDebugMode')==1, dbcont; end
---------
Not running simulation with the following parameters (already done): 
 Velocity: -15 
 Modes: 20 
 Version: 3 
---------
Starting simulation with velocity -25, modes 20, version v3 ... 
---------
Couldn't run simulation with the following parameters: 
 Velocity: -25 
 Modes: 20 
 Version: 3 
[Warning: The format 'yyyyMMddmmss' contains a field for minute (m) in what appears to be a date portion. You might have intended to
use the symbol for month (M) rather than for minute (m). See the <a href="matlab:doc('datetime.Format')">datetime.Format property</a> for a complete description of the
identifiers used in datetime formats.] 
[> In <a href="matlab:matlab.lang.internal.introspective.errorDocCallback('verifyFormat', '/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/private/verifyFormat.m', 34)" style="font-weight:bold">verifyFormat</a> (<a href="matlab: opentoline('/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/private/verifyFormat.m',34,0)">line 34</a>)
In <a href="matlab:matlab.lang.internal.introspective.errorDocCallback('datetime/subsasgnDot', '/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/subsasgnDot.m', 25)" style="font-weight:bold">datetime/subsasgnDot</a> (<a href="matlab: opentoline('/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/subsasgnDot.m',25,0)">line 25</a>)
In <a href="matlab:matlab.lang.internal.introspective.errorDocCallback('datetime/subsasgn', '/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/subsasgn.m', 23)" style="font-weight:bold">indexing</a> (<a href="matlab: opentoline('/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/subsasgn.m',23,0)">line 23</a>)
In <a href="matlab:matlab.lang.internal.introspective.errorDocCallback('sweeper_experiments', '/Users/eaguerov/Documents/Brown/impacts/km-dropplet-solidsubstrate-v3/matlab/1_code/sweeper_experiments.m', 122)" style="font-weight:bold">sweeper_experiments</a> (<a href="matlab: opentoline('/Users/eaguerov/Documents/Brown/impacts/km-dropplet-solidsubstrate-v3/matlab/1_code/sweeper_experiments.m',122,0)">line 122</a>)
] 
---------
Starting simulation with velocity -35, modes 20, version v3 ... 
---------
Couldn't run simulation with the following parameters: 
 Velocity: -35 
 Modes: 20 
 Version: 3 
[Warning: The format 'yyyyMMddmmss' contains a field for minute (m) in what appears to be a date portion. You might have intended to
use the symbol for month (M) rather than for minute (m). See the <a href="matlab:doc('datetime.Format')">datetime.Format property</a> for a complete description of the
identifiers used in datetime formats.] 
[> In <a href="matlab:matlab.lang.internal.introspective.errorDocCallback('verifyFormat', '/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/private/verifyFormat.m', 34)" style="font-weight:bold">verifyFormat</a> (<a href="matlab: opentoline('/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/private/verifyFormat.m',34,0)">line 34</a>)
In <a href="matlab:matlab.lang.internal.introspective.errorDocCallback('datetime/subsasgnDot', '/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/subsasgnDot.m', 25)" style="font-weight:bold">datetime/subsasgnDot</a> (<a href="matlab: opentoline('/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/subsasgnDot.m',25,0)">line 25</a>)
In <a href="matlab:matlab.lang.internal.introspective.errorDocCallback('datetime/subsasgn', '/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/subsasgn.m', 23)" style="font-weight:bold">indexing</a> (<a href="matlab: opentoline('/Applications/MATLAB_R2024a.app/toolbox/matlab/datatypes/datetime/@datetime/subsasgn.m',23,0)">line 23</a>)
In <a href="matlab:matlab.lang.internal.introspective.errorDocCallback('sweeper_experiments', '/Users/eaguerov/Documents/Brown/impacts/km-dropplet-solidsubstrate-v3/matlab/1_code/sweeper_experiments.m', 122)" style="font-weight:bold">sweeper_experiments</a> (<a href="matlab: opentoline('/Users/eaguerov/Documents/Brown/impacts/km-dropplet-solidsubstrate-v3/matlab/1_code/sweeper_experiments.m',122,0)">line 122</a>)
] 
Python 3.9.6
Traceback (most recent call last):
  File "/Users/eaguerov/Documents/Brown/impacts/km-dropplet-solidsubstrate-v3/matlab/1_code/sending_email.py", line 15, in <module>
    with open(path, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/eaguerov/Documents/Brown/impacts/km-dropplet-solidsubstrate-v3/matlab/0_data/credentials/lol.txt'
